com.github.bilelmoussaoui.Authenticator.json

```diff
@@ -0,0 +1,122 @@
+ {
+   "app-id": "com.github.bilelmoussaoui.Authenticator",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "3.26",
+   "sdk": "org.gnome.Sdk",
+   "command": "authenticator",
+   "finish-args": [
+     /* X11 + XShm */
+     "--share=ipc", "--socket=x11",
+     /* Wayland */
+     "--socket=wayland",
+     /* Filesystem */
+     "--filesystem=~/.config/Authenticator:ro",
+     /* Keyring */
+     "--talk-name=org.freedesktop.secrets",
+     /* dconf */
+     "--filesystem=xdg-run/dconf", "--filesystem=~/.config/dconf:ro",
+     "--talk-name=ca.desrt.dconf", "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+   ],
+   "build-options": {
+     "cflags": "-O2 -g",
+     "cxxflags": "-O2 -g",
+     "env": {
+       "V": "1"
+     }
+   },
+   "modules": [{
+       "name": "zbar",
+       "config-opts": [
+         "--without-qt",
+         "--without-gtk",
+         "--without-imagemagick",
+         "--disable-video"
+       ],
+       "sources": [{
+         "type": "archive",
+         "url": "http://downloads.sourceforge.net/project/zbar/zbar/0.10/zbar-0.10.tar.bz2",
+         "sha256": "234efb39dbbe5cef4189cc76f37afbe3cfcfb45ae52493bfe8e191318bdbadc6"
+       }]
+     },
+     {
+       "name": "pyotp",
+       "buildsystem": "simple",
+       "build-commands": [
+         "python3 setup.py install --prefix=/app --root=/"
```

Comment:
`--root=/` should be redundant for all of these.

Reply:
Fixed